src/components/hint/README.njk

```diff
@@ -14,79 +14,5 @@
 {% set componentGuidanceLink = false %}
 
 {% block componentArguments %}
-{{ govukTable({
-  'firstCellIsHeader': true,
-  'head' : [
-    {
-      text: 'Name'
-    },
-    {
-      text: 'Type'
-    },
-    {
-      text: 'Required'
-    },
-    {
-      text: 'Description'
-    }
-  ],
-  'rows' : [
-    [
-      {
-        text: 'text or (html) html'
-      },
-      {
-        text: 'string'
-      },
-      {
-        text: 'Yes'
-      },
-      {
-        text: 'Text or HTML to use within the hint. If `html` is provided, the `text` argument will be ignored.'
-      }
-    ],
-    [
-      {
-        text: 'id'
-      },
-      {
-        text: 'string'
-      },
-      {
-        text: 'Yes'
-      },
-      {
-        text: 'Optional id attribute o add to the hint span tag.'
-      }
-    ],
-    [
-      {
-        text: 'classes'
-      },
-      {
-        text: 'string'
-      },
-      {
-        text: 'No'
-      },
-      {
-        text: 'Optional additional classes to add to the hint span tag.'
-      }
-    ],
-    [
-      {
-        text: 'attributes'
-      },
-      {
-        text: 'object'
-      },
-      {
-        text: 'No'
-      },
-      {
-        text: 'Any extra HTML attributes (for example data attributes) to add to the hint span tag.'
-      }
-    ]
-  ]
-})}}
+See <a href="https://design-system.service.gov.uk/components/file-upload/#options-example-default--hint">options table</a> for details.
```

Comment:
I feel like the most vanilla hint would be using https://design-system.service.gov.uk/components/text-input/#options-example-default--hint but really doesnt matter.